packages/amplication-client/src/Purchase/FAQ.tsx

```diff
@@ -113,6 +113,32 @@ export const FAQ = () => {
             </div>
           }
         />
+        <Question
+          question="Can I pay for my subscription through AWS Marketplace?"
+          answer={
+            <div>
+              Yes, absolutely! We understand that many businesses prefer the
+              convenience and security of managing their software subscriptions
+              through AWS Marketplace. That's why we've made it possible for you
+              to subscribe to our services directly from there.
+              <br />
+              <br />
```

Comment:
Please avoid using `<br/>`. Use another HTML block element instead

Reply:
Hey @joyal007 just tried to do a check for quality assurance sake, seems like you're on the the right track, but the <p> elements might cause the bigger space between the divider and the answer:

example:
![image](https://github.com/amplication/amplication/assets/73097785/b6e94ffd-62a7-4858-9054-1edb83ba4c9f)